index.js

```diff
@@ -8,7 +8,8 @@ function getHeaders(method, resource, payload) {
   return {
     'Authorization': HmacAuthorize.sign(method, resource, 'application/json',
       payload ? JSON.stringify(payload) : null),
-    'User-Agent': 'ClassyPay Node.JS'
+    'User-Agent': 'ClassyPay Node.JS',
+    'Content-Type': 'application/json'
```

Comment:
This will work if we do this

```javascript
'Content-Type': payload ? 'application/json' : undefined
```

Reply:
Okay, I committed changes for this.

I did test the original changes with the `get`, `del`, `put`, and `post` methods and didn't run into authorization issues.